File: Logical Problems/reverse-number/reverse-number.cs
// C# program to reverse a number
using System;

class GFG {
	// Iterative function to
	// reverse digits of num
	static int reverseDigits(int num)
	{
		int rev_num = 0;
		while (num > 0) {
			rev_num = rev_num * 10 + num % 10;
			num = num / 10;
		}
		return rev_num;
	}

	// Driver code
	public static void Main()
	{
		int num = 4562;
		Console.Write("Reverse of no. is "
					+ reverseDigits(num));
	}
}

// This code is contributed by Sam007

File: Logical Problems/reverse-number/reverse-number.php
<?php
// Iterative function to
// reverse digits of num
function reverseDigits($num)
{
	$rev_num = 0;
	while($num > 1)
	{
		$rev_num = $rev_num * 10 +
						$num % 10;
		$num = (int)$num / 10;
	}
	return $rev_num;
}

// Driver Code
$num = 4562;
echo "Reverse of no. is ",
	reverseDigits($num);

// This code is contributed by aj_36
?>
